src/checkpoint/dispatch/dispatch.h

```diff
@@ -188,21 +188,21 @@ struct Standard {
   static SerialByteType* allocate();
 };
 
-template <typename T>
+template <typename T, typename... UserTraits>
 buffer::ImplReturnType packBuffer(
   T& target, SerialSizeType size, BufferObtainFnType fn
 );
 
 template <typename Serializer, typename T>
 inline void serializeArray(Serializer& s, T* array, SerialSizeType const len);
```

Comment:
should this have UserTraits also?

Reply:
It should not have. It still have UserTraits, but now for good reason